ui/app/AppLayouts/Profile/views/SyncView.qml

```diff
@@ -22,6 +22,15 @@ Item {
 
     Layout.fillHeight: true
     Layout.fillWidth: true
+    
+    property string activeMailserver: ""
```

Comment:
@richard-ramos continuation to the previous comment... please revert all changes done in this qml file, as `activeMailserver` is update and it's a property which will be notified about the change just sending appropriate signal from the backend part.